src/Dfe.PlanTech.DatabaseUpgrader/Scripts/2024/20240729_15:58_AddCSLinkToRecommendationChunk.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE Contentful.RecommendationChunks
+    ADD [CSLinkText] NVARCHAR(256),
+        [CSURL] NVARCHAR(256);
```

Comment:
two tiny things
this rename keeps it consistent with rest of the db
```suggestion
        [CSUrl] NVARCHAR(256);
```
and filename needs the `15:58` swapping to `1558`

Reply:
[79c3dfe](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/718/commits/79c3dfe3847af48f2a9bb99c54f9d0e7137c7df6)